### ISSUE TITLE
Binding a KVar to an input element's value and typing causes an exception

### ISSUE THREAD

**Issue by chandrin94** [kwebio/kweb-core#186]
**Describe the bug**
Hello,

i noticed that since version 0.8.6, binding a KVar to an input element's value (e.g a textArea in my case) and then trying to type in that input element throws an JSonSyntaxException.

**To Reproduce**
To reproduce this behaviour, just bind a KVar to an input element like explained in 
http://docs.kweb.io/en/latest/state.html?highlight=kvar#binding-a-kvar-to-an-input-element-s-value
and then type inside the input element.

**Expected behavior**
Changes in the browser by the user should be reflected immediately in the KVar.

**Actual behaviour**
An exception with the following stackTrace is thrown:

```
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected a string but was BEGIN_OBJECT at line 1 column 15 path $.retrieved
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:224)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	at com.google.gson.Gson.fromJson(Gson.java:852)
	at com.google.gson.Gson.fromJson(Gson.java:801)
	at kweb.ValueElement$setValue$$inlined$event$1.invoke(OnReceiver.kt:163)
	at kweb.ValueElement$setValue$$inlined$event$1.invoke(OnReceiver.kt:14)
	at kweb.html.events.OnReceiver$event$1.invoke(OnReceiver.kt:17)
	at kweb.html.events.OnReceiver$event$1.invoke(OnReceiver.kt:14)
	at kweb.Element$addEventListener$1.invoke(Element.kt:347)
	at kweb.Element$addEventListener$1.invoke(Element.kt:24)
	at kweb.Kweb.listenForWebsocketConnection(Kweb.kt:312)
	at kweb.Kweb$listenForWebsocketConnection$1.invokeSuspend(Kweb.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalStateException: Expected a string but was BEGIN_OBJECT at line 1 column 15 path $.retrieved
	at com.google.gson.stream.JsonReader.nextString(JsonReader.java:825)
	at com.google.gson.internal.bind.TypeAdapters$16.read(TypeAdapters.java:418)
	at com.google.gson.internal.bind.TypeAdapters$16.read(TypeAdapters.java:406)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
Caused by: java.lang.IllegalStateException: Expected a string but was BEGIN_OBJECT at line 1 column 15 path $.retrieved

	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
```

**Comment by sanity** [comment]
Looks like a duplicate of #181

**Comment by sanity** [comment]
Should be fixed in `0.9.0`.
